### DISCUSSION TITLE
GET /api/websites/:websiteId/stats returns 0 for everything

### DISCUSSION THREAD

**Discussion by krumjahn** [umami-software/umami#3020] [Q&A]
First of all, I'm new and I love umami.

<img width="1295" alt="Screenshot 2024-10-23 at 4 16 13 PM" src="https://github.com/user-attachments/assets/e713c484-26e3-476c-80fd-7e502fada25e">

- I'm hosting it on Synology NAS using portainer.
- I have umami running and I see the stats in my dashboard. 
- I got my API token
- I am able to call website ID's and get all my IDs
- I want to get the stats of each website. I'm calling GET /api/websites/:websiteId/stats

I've tried all sorts of time formats but all of them return 0. 

curl -X GET 'https://<mywebsite>/api/websites/f53cf938-91aa-4875-85d8-bd89f9c9da91/stats?startAt=1697616199434&endAt=1698076819687'

I've tried looking at past 4 hours, past 2 days, past 30 days. None of it gives me any stats. All I get is:
{"pageviews":{"value":0,"prev":0},"visitors":{"value":0,"prev":0},"visits":{"value":0,"prev":0},"bounces":{"value":0,"prev":0},"totaltime":{"value":0,"prev":0}}

Any help would be appreciated!!

**Comment by julianwitzel** [comment]
Did you fix this? For me it returns 400 status codes every time I want some stats. I can get teams and pages but no stats for some reason.

**Reply by smdp2000** [reply]
even im getting all zeros
